src/main/java/org/jboss/xavier/integrations/jpa/service/WorkloadInventoryReportService.java

```diff
@@ -49,6 +49,23 @@
         return reportRepository.findByAnalysisOwnerAndAnalysisId(analysisOwner, analysisId);
     }
 
+    public List<WorkloadInventoryReportModel> findByAnalysisOwnerAndAnalysisId(
```

Comment:
I can't see any Test testing the body of this method.

Reply:
Added a test